doc/TRANSLATORS.md

```diff
@@ -1,82 +1,113 @@
+1. Get the PO file for your language from the repository -> [po](../po)
 
-    Then you will be asked to save the newly created PO file. Give it a name like fr.po for French translation, de.po for German translation and so on. If you are not sure about the two-letter code for your language, use the one that local websites in your native language use.
+2. To support making release translations easier, the darktable build process automatically creates a PO template (.POT) file, `darktable.pot` to use as base.
 
-4. In the PO editor you will see a list of all translatable messages, a field that contains an original message in English and a field to input your translation. The best practice is to copy the original message into translation field and then translate all the translatable bits. That way you won't forget about some markup we use. It's also recommended in poedit to go to "Edit > Preferences" and in the "General" tab check the "Always change focus to text input field" checkbox.
+3. Get the latest [darktable.pot](../po/darktable.pot) file.
 
-5. The best practice is to frequently test your changes. To do this, first open the LINGUAS file with your favorite text editor(not poedit) and add the two-letter code for your language there. Save, quit. Now you only have to run `make` and `sudo make install` again and restart darktable to see your changes.
+4. Put both files on the same directory. Open the po file with a po file editor, e.g. poedit:
+    ```bash
+    $ poedit <lang>.po
+    ```
+5. Update the PO file using the POT file: go to menu `[Catalog]` ->
+`[Update from POT file...]` and select `darktable.pot` file.
 
-6. If you don't understand some terminology, refer to existing translations from UFRaw or Rawstudio or digiKam or ask in the mailing list.
+6. Start translating. Saving will update `<lang>.po`.
 
-7. When you are done, `bzip2` your PO file and send it to the mailing list.
+7. When you are done, `bzip2` your PO file and send it to the [mailing list](https://www.darktable.org/contact/) or per instructions on the [wiki](https://github.com/darktable-org/darktable/wiki/Translations).
 
-Now about updates. Every new version of darktable contains new functionality and thus there are always new messages to translate. It's best to keep track of changes, but even so we announce strings freeze a few days before releasing. It means that no changes to translatable messages will be added, so your translation created during that period will work fine.
+8. That's all!
 
-To update your translation from working your tree:
+## Translate the darktable User Interface for darktable releases (how-to, git)
```

Comment:
For darktable releases we use the committed pot file as described above. I would completely remove this section. At this stage translators should not care about Git at all.

Reply:
Made changes as requested.